### DISCUSSION TITLE
Build can't find some artifacts

### DISCUSSION THREAD

**Discussion by castortech** [infinispan/infinispan#10865] [Q&A] [answered]
I am trying to build Infinispan and getting an error about some artifacts that can't be found.

`[ERROR] Failed to execute goal on project infinispan-cdi-embedded: Could not resolve dependencies for project org.infinispan:infinispan-cdi-embedded:jar:14.0.9-SNAPSHOT: The following artifacts could not be resolved: org.apache.taglibs:taglibs-standard-spec:jar:1.2.6-RC1 (present, but unavailable), org.apache.taglibs:taglibs-standard-impl:jar:1.2.6-RC1 (present, but unavailable): org.apache.taglibs:taglibs-standard-spec:jar:1.2.6-RC1 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
`

From what I was able to see these might exist in a JBoss repository but not on central. 

Is there anything special that must be done just to build successfully?

Thanks
Alain

**Comment by tristantarrant** [comment, marked as answer]
`mvn -s maven-settings.xml ...`

**Reply by castortech** [reply]
Thanks Tristan
